data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
 		"drag" 16.1
 		"heat dissipation" .4
 		"fuel capacity" 700
-		"cargo space" 530
-		"outfit space" 740
-		"weapon capacity" 300
-		"engine capacity" 180
+		"cargo space" 240
```

Comment:
>With 240, I think it would affect my ability to be even only a warship too greatly.

As I said above, this change puts its max outfit capacity to 900. That's either equal to or more than all other human heavy warships, so I don't see how it's a problem as is.